### PR TITLE
Adding JSON files for Prometheus queries for Runtimes.

### PR DESCRIPTION
**Pull request by KillianGolds**
## Description

[RHOAIENG-7353](https://issues.redhat.com/browse/RHOAIENG-7353)
[RHOAIENG-3052](https://issues.redhat.com/browse/RHOAIENG-3052)
Prereq for #204 

## How Has This Been Tested?

I tested each query against a deployment of each runtime mentioned in the commits. To ensure the correct metric is returned.

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by VedantMahabaleshwarkar** [comment]
@alexcreasy could you take a look at this PR and lgtm purely in terms of the structure of the json files and ensuring that it looks like what has been discussed.

File: controllers/constants/ovms-metrics.json
Comment (alexcreasy) [inline controllers/constants/ovms-metrics.json:3]: @VedantMahabaleshwarkar you mentioned this needed to be a string value rather than a boolean for technical reasons - is that the case? The dashboard is currently looking for string "true" || "false".

File: controllers/constants/ovms-metrics.json
Comment (alexcreasy) [inline controllers/constants/ovms-metrics.json:44]: @VedantMahabaleshwarkar Have you got these titles OK'd with UXD? They look extremely long and if we're moving to a model of displaying these strings directly in the dashboard they have to be approved.

This point doesn't need to hold up this PR but it's something that will need to be taken care of before it can be signed off.

File: controllers/constants/ovms-metrics.json
Comment (KillianGolds) [reply controllers/constants/ovms-metrics.json]: Hi Alex, updated the bool to be a string in the second commit! It was an oversight on my part. Apologies.

File: controllers/constants/ovms-metrics.json
Comment (KillianGolds) [reply controllers/constants/ovms-metrics.json]: Hi Alex, matched the titles to those that would be displayed by modelmesh in the second commit. Let me know if this is more suitable!

**Review by VedantMahabaleshwarkar — APPROVED** [review]
/lgtm

**Pull request merged**
